<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="showImage">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>14</y>
      <width>775</width>
      <height>665</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseImageFolder">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>690</y>
      <width>356</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>select images</string>
    </property>
   </widget>
   <widget class="QLabel" name="showInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="extractFeat">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>740</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract library features</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>700</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>One sentence to describe the picture you want to search (only supports English)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetText">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>760</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>810</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
